src/main/scala/com/lightbend/paradox/apidoc/ApidocPlugin.scala

```diff
@@ -50,10 +50,11 @@ object ApidocPlugin extends AutoPlugin {
           val scanner = new ClassGraph()
             .whitelistPackages(apidocRootPackage.value)
             .addClassLoader(classLoader)
+            .enableMethodInfo()
```

Comment:
I wonder what the impact of this on the running time is - classpath scanning is already a noticable delay in the documentation generation of Akka, does also including method info add to that much?

Reply:
For me, it looks like it take 1-2s longer now.